### PR TITLE
Sentiment analysis on tweets

### PR DESCRIPTION
**Pull request by ghost**
This sentiment analysis uses sets of negative and positive words to compute a weighted sentiment score on each tweet, giving each a rating equal to the following: `NEUTRAL`, `NEGATIVE` or `POSITIVE`.

### CODE REVIEW

File: src/main/scala/utils/SentimentAnalysisUtils.scala [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    val negWordWeight = countWeight(tokens, negWordsSet)
+
+    println("DEBUG - pos weight: " + posWordWeight)
+    println("DEBUG - neg weight: " + negWordWeight)
+
+    val sentiment = posWordWeight - negWordWeight
+
+    println("DEBUG - sentiment score = " + sentiment)
+
+    sentiment match {
```
Comment (eftov) [inline src/main/scala/utils/SentimentAnalysisUtils.scala]: Codacy Issue found: [Default case not specified.](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1633170999/issues/source?bid=2980032&fileBranchId=3034495#l66)

File: src/main/scala/utils/SentimentAnalysisUtils.scala [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  }
+
+  def loadWordSet(path: String) = {
+    val stream: InputStream = getClass.getResourceAsStream(path)
+    val lines = fromInputStream(stream).getLines
+    lines.toSet
+  }
+
+  def countWeight(tokens: Seq[String], wordSet: Set[String]): Int = {
+    var weight = 0
```
Comment (eftov) [inline src/main/scala/utils/SentimentAnalysisUtils.scala]: Codacy Issue found: [Avoid mutable local variables](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1633170999/issues/source?bid=2980032&fileBranchId=3034495#l38)

File: src/test/scala/SentimentAnalysisUtilsTest.scala [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import org.scalatest.FunSpec
+import org.scalatest.matchers.ShouldMatchers
+import utils.SentimentAnalysisUtils._
+
+class SentimentAnalysisUtilsTest extends FunSpec with ShouldMatchers{
+  describe("A sentiment analysis utility class") {
+    it ("should detect a negative statement") {
+      detectSentiment("@Sosh_fr iPhone volé, je fais quoi maintenant ?") should equal (NEGATIVE)
```
Comment (eftov) [inline src/test/scala/SentimentAnalysisUtilsTest.scala]: Codacy Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1633171000/issues/source?bid=2980032&fileBranchId=3034495#l8)

Comment (eftov) [inline src/test/scala/SentimentAnalysisUtilsTest.scala]: Codacy Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1663471696/issues/source?bid=2980032&fileBranchId=3034495#l8)

File: src/main/scala/utils/SentimentAnalysisUtils.scala [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    val negWordWeight = countWeight(tokens, negWordsSet)
+
+    println("DEBUG - pos weight: " + posWordWeight)
+    println("DEBUG - neg weight: " + negWordWeight)
+
+    val sentiment = posWordWeight - negWordWeight
+
+    println("DEBUG - sentiment score = " + sentiment)
+
+    sentiment match {
```
Comment (eftov) [inline src/main/scala/utils/SentimentAnalysisUtils.scala]: Codacy Issue found: [Default case not specified.](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1663471695/issues/source?bid=2980032&fileBranchId=3034495#l66)

File: src/main/scala/utils/SentimentAnalysisUtils.scala (hunk 2 above)
Comment (eftov) [inline src/main/scala/utils/SentimentAnalysisUtils.scala]: Codacy Issue found: [Avoid mutable local variables](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1663471695/issues/source?bid=2980032&fileBranchId=3034495#l38)

Comment (ghost) [inline src/main/scala/utils/SentimentAnalysisUtils.scala:38]: Codacy Issue found: [Avoid mutable local variables](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1663539364/issues/source?bid=2980032&fileBranchId=3034495#l38)

File: src/main/scala/utils/SentimentAnalysisUtils.scala (hunk 4 above)
Comment (ghost) [inline src/main/scala/utils/SentimentAnalysisUtils.scala:66]: Codacy Issue found: [Default case not specified.](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1663539364/issues/source?bid=2980032&fileBranchId=3034495#l66)

File: src/test/scala/SentimentAnalysisUtilsTest.scala (hunk 3 above)
Comment (ghost) [inline src/test/scala/SentimentAnalysisUtilsTest.scala:8]: Codacy Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1663539366/issues/source?bid=2980032&fileBranchId=3034495#l8)

**Pull request merged**